### PR TITLE
MMCA 4542 - Update version, configuration & warnings for CDS Financials

### PR DESCRIPTION
**Pull request by HariHmrc**

### CODE REVIEW

File: app/views/helpers/Formatters.scala [hunk 1]
```diff
@@ -68,7 +68,7 @@ trait CurrencyFormatters {
 
 trait FileFormatters {
   def fileSize(size: Long): String = size match {
-    case kb if 1000 until 1000000 contains kb => s"${kb / 1000}KB"
+    case kb if kb >= 1000 && kb < 1000000 => s"${kb / 1000}KB"
```
Comment (AdamCoops) [inline app/views/helpers/Formatters.scala:71]: Can we make these values some private vans so they're not magic numbers

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -2,22 +2,23 @@ import sbt.*
...
-    "uk.gov.hmrc" %% "play-conditional-form-mapping" % "1.13.0-play-28",
-    "org.typelevel" %% "cats-core" % "2.9.0",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "1.3.0",
-    "uk.gov.hmrc" %% "emailaddress" % "3.8.0"
+    play.sbt.PlayImport.ws,
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-30" % bootstrapVersion,
+    "uk.gov.hmrc" %% "play-frontend-hmrc-play-30" % "8.5.0",
+    "uk.gov.hmrc" %% "play-conditional-form-mapping-play-30" % "2.0.0",
+    "org.typelevel" %% "cats-core" % "2.10.0",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30" % "1.7.0",
```
Comment (AdamCoops) [inline project/AppDependencies.scala:13]: I think this can be 1.8.0

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -2,22 +2,23 @@ import sbt.*
...
-    "uk.gov.hmrc" %% "bootstrap-test-play-28" % bootstrapVersion % Test,
-    "org.jsoup" % "jsoup" % "1.16.1" % Test,
-    "com.vladsch.flexmark" % "flexmark-all" % "0.36.8" % "test, it",
-    "org.mockito" %% "mockito-scala-scalatest" % "1.17.14" % "test, it",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "1.3.0"
+    "uk.gov.hmrc" %% "bootstrap-test-play-30" % bootstrapVersion % Test,
+    "org.jsoup" % "jsoup" % "1.16.1",
+    "com.vladsch.flexmark" % "flexmark-all" % "0.64.8" % Test,
+    "org.mockito" %% "mockito-scala-scalatest" % "1.17.29" % Test,
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30" % "1.7.0"
```
Comment (AdamCoops) [inline project/AppDependencies.scala:22]: 1.8.0

File: project/plugins.sbt [hunk 4]
```diff
@@ -3,10 +3,12 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.20.0")
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.2.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.20")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.5.0")
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.1")
```
Comment (AdamCoops) [inline project/plugins.sbt:7]: Have you checked this against 3.0.2?

Comment (HariHmrc) [reply project/plugins.sbt:7]: Yes, its working with 3.0.2

**Pull request merged**